### ISSUE TITLE
Accessibility -  AA WCAG 2.5.8 Touch Target Size Issue

### ISSUE THREAD

**Issue by acovher2** [web-illinois/toolkit-builder-3#26]
**TEST ENVIRONMENT:**
Using the **'ilw-font' variant (preferred)**
https://builder3.toolkit.illinois.edu/demo/home-pure.html 

**DEVICE:**
Dell Windows PC 
**BROWSER:**
Chrome Browser Version 124.0.6367.93 (official build) (64-bit) 

Site Improve Chrome Extension automated test tool was used:
"**Description:**
Interactive element does not meet minimum size nor spacing
Interactive elements, like buttons or links, should be large enough or be positioned, so they can easily be clicked or tapped by users.

Elements should be at least 24 by 24 pixels, or if smaller, be positioned with enough spacing to not intersect with other interactive elements. An adequate size and positioning of interactive elements are important so people with physical impairments or using touchscreens can interact with them without difficulty."

This tool found 24 instances of this WCAG 2.5.8 Touch Target (AA) violation. 

**Screenshots:**
The image below has the violating elements within Red Rectangles.
<img width="960" alt="WCAG 2 5 8_ilw-font variant_builder3-toolkit-illinois-edu-demo-home-pure-html-2024-11-14-14_04_30" src="https://github.com/user-attachments/assets/8617d557-b970-49ca-8933-ccc4cc797e04">

Please let me know if you have any questions/comments.

**Comment by bryanjonker-illinois** [comment]
Talked with Amanda, and this can be closed
